src/Symfony/Component/Form/Resources/translations/validators.it.xlf

```diff
@@ -8,11 +8,131 @@
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Il valore non è un colore HTML5 valido.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Per favore, inserisci una data di compleanno valida.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>La scelta selezionata non è valida.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>La collezione non è valida.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Per favore, seleziona un colore valido.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Per favore, seleziona un paese valido.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Per favore, seleziona una valuta valida.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Per favore, scegli a valid date interval.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Per favore, inserisci a valid date and time.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Per favore, inserisci a valid date.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Per favore, seleziona un file valido.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Il campo nascosto non è valido.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Per favore, inserisci un numero intero.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Per favore, seleziona una lingua valida.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Per favore, seleziona una lingua valida.</target>
```

Comment:
Make sure this says "locale" and not language.

Reply:
We don’t have a word for “locale” in Italian. The word “locale” means “room” or “a place where to eat, dance or something similar”.

The word “lingua” means both “locale” and “language”.

We have the verb “localizzare”, that means “find something or someone in a space” and also means “translating”.

But the corresponding noun “locale” is not used with the meaning of a “local language”.

Reply:
Okey. Thank you. 

I was just making sure